Makefile

```diff
@@ -1,9 +1,11 @@
 # Some simple testing tasks (sorry, UNIX only).
 
+pytest := python3 -m pytest
```

Comment:
yes, with this change you can run the tests outside virtualenv, still want me revert?  it makes no different inside virtual env but fixes it outside virtual env